test/ngraph.cpp

```diff
@@ -29,7 +29,8 @@ using namespace std;
 TEST(NGraph, loadTest)
 {
     // load the triangle library
-    void* ngraphImplLib = dlopen("../src/libngraph.so", RTLD_LAZY);
+    void* ngraphImplLib = dlopen("../src/libngraph.so", RTLD_LAZY) ||
```

Comment:
We could do this the same way it is done in backend.cpp, that way it supports all targets.